README.md

```diff
@@ -96,32 +103,41 @@ You can pass arbitrary build flags into the build command with the `build_flags`
 }
 ```
 
-## Custom paths
+### Custom paths
 
 You can set the following paths:
 
-- `input_path`: path to your project. defaults to the current directory
-- `go_binary_path`: path to the `go` binary. defaults to "go"
-- `output_path`: path to the dir where you'll find the bundle results. defaults to `current directory/output`
-- `resources_path`: path where the `resources` dir is and will be written. path must be relative to the `input_path`. defaults to "resources"
-- `vendor_dir_path`: path where the `vendor` dir will be written. path must be relative to the `output_path`
-- `working_directory_path`: path to the dir where the bundler runs its operations such as provisioning the vendor files or binding data to the binary
+* `input_path`: path to your project. defaults to the current directory
+* `go_binary_path`: path to the `go` binary. defaults to "go"
+* `output_path`: path to the dir where you'll find the bundle results. defaults to `current directory/output`
+* `resources_path`: path where the `resources` dir is and will be written. path must be relative to the `input_path`. defaults to "resources"
+* `vendor_dir_path`: path where the `vendor` dir will be written. path must be relative to the `output_path`
+* `working_directory_path`: path to the dir where the bundler runs its operations such as provisioning the vendor files or binding data to the binary
 
-## Adapt the bind configuration
+### Adapt the bind configuration
 
 You can use the `bind` attribute to alter the bind configuration like so:
 
 ```json
 {
   "bind": {
     "output_path": "path/to/bind/output/path",
-    "package": "mypkg"
+    "package": "mypkg",
+    "package_path": "some/module/path/mypkg"
```

Comment:
Please update the doc based on my comments below

Reply:
Will do!